pilot/pkg/xds/eds.go

```diff
@@ -183,7 +186,7 @@ func (s *DiscoveryServer) edsCacheUpdate(shard model.ShardKey, hostname string,
 		Namespace: namespace,
 	}: {}})
 	ep.mutex.Unlock()
-	if features.SendUnhealthyEndpoints && pushType == IncrementalPush {
```

Comment:
Should elaborate on the flow under the bracket

Reply:
just below bracket there are comments